steps/github_api_steps.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python3
+# -*- coding: ascii -*-
+import logging
+import time
+from functools import wraps
+
+from requests.auth import HTTPBasicAuth
+
+from utils.http_client import HttpClient
+
+logger = logging.getLogger(__name__)
+
+
+def credentials_checkup(func):
+    @wraps(func)
+    def wrapped(obj, *args, **kwargs):
```

Comment:
Done. I was thinking that self is usually used within bound methods declaration (with the meaning of instance of this class). Because my decorator is defined outside of any class and is not bound I decided to use obj